internal/terraform/exec/exec_test.go

```diff
@@ -57,21 +62,29 @@ func TestExec_cancel(t *testing.T) {
 	t.Fatalf("expected cancel error: %#v, given: %#v", expectedErr, err)
 }
 
-func newExecutor(t *testing.T) TerraformExecutor {
-	tmpDir := TempDir(t)
-	workDir := filepath.Join(tmpDir, "workdir")
-	err := os.MkdirAll(workDir, 0755)
-	if err != nil {
-		t.Fatal(err)
-	}
-	installDir := filepath.Join(tmpDir, "installdir")
-	err = os.MkdirAll(installDir, 0755)
+func newExecutor(t *testing.T, tfVersion string) TerraformExecutor {
+	ctx := context.Background()
+	workDir := TempDir(t)
+	installDir := filepath.Join(workDir, "hcinstall")
+	err := os.MkdirAll(installDir, 0755)
 	if err != nil {
 		t.Fatal(err)
 	}
```

Comment:
```suggestion
	if err := os.MkdirAll(installDir, 0755); err != nil {
		t.Fatal(err)
	}
	t.Cleanup(func() {
		if err := os.Remove(installDir); err != nil {
			t.Fatal(err)
		}
	})
```

Reply:
`installDir` is under the `workDir`. It's created using the `TempDir()` function which registers a cleanup function the will delete the whole `workDir`. Do you want to have another cleanup function anyway?

Reply:
See my comment above.